jujugui/static/gui/src/app/components/revenue-statement/revenue-statement.js

```diff
@@ -0,0 +1,255 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+
+const Spinner = require('../spinner/spinner');
+const BasicTable = require('../basic-table/basic-table');
+
+/**
+  React component used to display a list of the users revenue-statements in their profile.
+*/
+class RevenueStatement extends React.Component {
+  constructor() {
+    super();
+    this.xhrs = [];
+    this.state = {
+      data: [
```

Comment:
I'm guessing this is dummy data? Can you comment to that effect that it needs to be removed.